Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/ui/details/PersonDetailsFragment.kt

```diff
@@ -84,6 +90,12 @@ class PersonDetailsFragment : Fragment(R.layout.person_details_fragment), AutoIn
         }
     }
 
+    private fun navigateBack() {
+        doNavigate(
+            PersonDetailsFragmentDirections.actionPersonDetailsFragmentToPersonOverviewFragment()
```

Comment:
this makes the user needs to scroll again to where he was , can we improve that?

Reply:
And leads to navigation loop 🔁  by clicking the back button 
PersonDetailsFragment <---> PersonOverviewFragment

Reply:
Not sure what else we can do about it. When you get to the PersonOverviewFragment you can access the items on the bottom nav so you could get out of this navigation loop. If we don't do this here, you'd get stuck in a loop between the PersonDetailsFragment and the certificate details fragment. The navigation stack gets broken by traveling back from the Certificate Details Fragment to the PersonDetailsFragment instead of the PersonOverviewFragment so I've tried to mitigate it as much as possible. I could force the back button to take you to the HomeFragment, but what if the user traveled from the Tagebuch or the Check-In fragments to the PersonOverviewFragment? The loop wouldn't exist anymore but the navigation would still be wrong.

Reply:
To avoid these cases I followed another approach in https://github.com/corona-warn-app/cwa-app-android/pull/3536
Please have a look ? and let me know what do you think ?

Reply:
Yes, that's much more elegant indeed 👍